### PR TITLE
Check for mstatus.FS when performing half-precision loads/stores

### PR DESCRIPTION
**Pull request by arichardson**
It seems inconsistent to me that half-precision loads/stores are allowed if mstatus.FS is set to 0 but single/double-precision ones are illegal.

### CODE REVIEW

**Comment by arichardson** [comment]
ping? Would be good to know if this is expected behaviour or an oversight.

File: model/riscv_sys_regs.sail
Comment (jrtc27) [inline model/riscv_sys_regs.sail:351]: This also needs `(misa.F() == 0b1)`; Zfh depends on F.

Comment (arichardson) [reply model/riscv_sys_regs.sail]: Done, sorry for the delay.

**Comment by arichardson** [comment]
ping? Is there anything holding this back? It would be great if PRs could be merged or if there was some indication on what is blocking it from being merged.

The current wait time towards getting even trivial changes merged makes me wonder if I should be spending my time on other things rather than the sail model.

**Comment by jrtc27** [comment]
As far as I can tell there are no maintainers who have experience with managing a collaborative open-source software project, so ending up in this situation is hardly surprising. The weekly meetings seem to focus on specifics whilst ignoring the larger problem of:
1. a lack of people who know how to review code
2. a lack of people who make sure pull requests are merged in a timely manner
Perversely, the situation was better when Cambridge had ownership of the model, because then we could just merge things ourselves (including from outside contributors). Now that it's owned by RISC-V International there's a systematic problem with a lack of ownership for getting pull requests properly reviewed and merged.

Maybe I should just make use of the fact that I have write permission to the repository and merge this myself, even though I have no clue what my official status is and what the official policy is for me doing that. The current situation is ridiculous.

**Review by ptomsich — APPROVED** [review]
LGTM.

**Comment by ptomsich** [comment]
For the lack of an official policy (and with one approving review), I'll just use my permissions on this repository to merge.

**Comment by arichardson** [comment]
@ptomsich thank you so much for going ahead and merging/checking up on all the open PRs!

**Pull request merged**
